x/ibc/02-client/client/utils/utils.go

```diff
@@ -80,13 +80,13 @@ func QueryClientStateABCI(
 		return nil, nil, 0, err
 	}
 
-	proofBz, err := clientCtx.Codec.MarshalBinaryBare(res.Proof)
+	proofBz, err := clientCtx.LegacyAmino.MarshalBinaryBare(res.Proof)
```

Comment:
I'm going to address this in a followup pr since I'm not entirely sure how to interact with proto encoding/decoding through the context atm. This is incorrect all through IBC client/ so not necessary for the scope of this pr